enterprise/dev/ci/internal/ci/operations.go

```diff
@@ -688,3 +688,11 @@ func publishExecutorDockerMirror(version string) operations.Operation {
 		pipeline.AddStep(":packer: :white_check_mark: docker registry mirror image", stepOpts...)
 	}
 }
+
+func buildeventUploadTrace() operations.Operation {
```

Comment:
nit - more inline with the script and verb first reads a little smoother:

```suggestion
func uploadBuildeventTrace() operations.Operation {
```